node_modules/oae-activity/tests/test-email.js

```diff
@@ -855,6 +857,59 @@ describe('Activity Email', function() {
     });
 
     /**
+     * Test that verifies that activity emails are not delivered to users who have deleted their
+     * profiles
+     */
+    it('verify email is not delivered to users who have since deleted their profile', function(callback) {
```

Comment:
> Can you describe the potential performance drawbacks that are involved here?

The draw back is that every time you expand the authz memberships of a group you need to run the ids through `AuthzDeleted.isDeleted` (cassandra multi-get). So that would be permissions checks, activity routing, etc...